GTG/gtk/editor/taskeditor.ui

```diff
@@ -4,403 +4,461 @@
-                    <property name="can_focus">True</property>
-                    <property name="resize_mode">immediate</property>
-                    <property name="wrap_mode">word</property>
-                    <property name="left_margin">5</property>
-                    <property name="right_margin">5</property>
-                  </object>
-                </child>
+                <property name="icon-name">open-menu-symbolic</property>
+                <property name="icon-size">1</property>
               </object>
-              <packing>
-                <property name="expand">True</property>
-                <property name="fill">True</property>
-                <property name="position">0</property>
-              </packing>
             </child>
           </object>
           <packing>
-            <property name="expand">True</property>
-            <property name="fill">True</property>
-            <property name="position">1</property>
-          </packing>
-        </child>
-        <child>
-          <object class="GtkSeparator" id="hseparator1">
-            <property name="visible">True</property>
-            <property name="can_focus">False</property>
-          </object>
-          <packing>
-            <property name="expand">False</property>
-            <property name="fill">False</property>
-            <property name="position">2</property>
+            <property name="pack_type">end</property>
           </packing>
+          <style>
+            <class name="image-button"/>
+          </style>
         </child>
+      </object>
+    </child>
+    <child>
+      <object class="GtkBox" id="vbox4">
+        <property name="visible">True</property>
+        <property name="can_focus">False</property>
+        <property name="orientation">vertical</property>
         <child>
-          <object class="GtkBox" id="tasksidebar">
+          <object class="GtkBox">
+            <property name="margin">3</property>
             <property name="visible">True</property>
-            <property name="can_focus">False</property>
-            <child>
-              <object class="GtkLabel" id="label2">
-                <property name="visible">True</property>
-                <property name="can_focus">False</property>
-                <property name="xalign">0</property>
-                <property name="label" translatable="yes">Starts on</property>
-                <attributes>
-                  <attribute name="weight" value="bold"/>
-                </attributes>
-              </object>
-              <packing>
-                <property name="expand">False</property>
-                <property name="fill">True</property>
-                <property name="padding">10</property>
-                <property name="position">0</property>
-              </packing>
-            </child>
             <child>
-              <object class="GtkBox" id="box1">
+              <object class="GtkBox" id="task_toolbar_box">
```

Comment:
Good news, bad news! Good news, I found a solution for this issue. Put those buttons in a separate GtkBox and assign it style `linked`:

```xml
<object class="GtkBox">
  <property name="visible">True</property>
  <style>
    <class name="linked"/>
  </style>
  <child>
    <object class="GtkButton" id="bold">
```

Found this in [gnome-builder ui](https://github.com/chergert/gnome-builder/blob/master/data/ui/gb-view-stack.ui#L192-L199)

Result:
![button groups](https://cloud.githubusercontent.com/assets/98588/8504451/fad5e524-21d1-11e5-8a07-ecb16323b526.png)

This is second time I've burned myself on Gtk styles :-1: I found there is a list of all styles in [GtkStyleContext](https://developer.gnome.org/gtk3/stable/GtkStyleContext.html)

Bad news: it doesn't have any common with GtkButtonBox. Sorry for leading you onto a wrong path to solve this :cold_sweat: